gslib/gcs_json_api.py

```diff
@@ -228,7 +231,18 @@ def __init__(self,
 
     self.http_base = 'https://'
     gs_json_host = config.get('Credentials', 'gs_json_host', None)
-    self.host_base = gs_json_host or 'storage.googleapis.com'
+    if context_config.get_context_config(
+    ) and context_config.get_context_config().use_client_certificate:
```

Comment:
Are you missing test for this section? I think we can simulate the context config and add a test to make sure this error gets raised?

Reply:
Wasn't sure if I should cross the threshold of creating a new test file for this untested class.
Done.